packages/tab/src/index.d.ts

```diff
@@ -0,0 +1,44 @@
+import { FunctionComponent, ReactChild, Component, MouseEvent } from 'react'
+
+interface Styling {
```

Comment:
Yeah, I think this is a good idea to separate.  Eventually, perhaps even more general, because these are meant to be applicable all UI components.